frontend/simple/js/backend/hapi.js

```diff
@@ -95,6 +95,11 @@ export class HapiBackend extends Backend {
     store.dispatch('saveSettings')
     return res
   }
+  postEvent (event, contract) {
+    return request.post(`${process.env.API_URL}/event/${contract || event.toHash()}`)
+      .set('Authorization', `gi ${signature}`)
+      .send({hash: event.toHash(), entry: event.toObject()})
+  }
```

Comment:
This needs to be removed because this functionality is already implemented by publishLogEntry.